pkg/comp-functions/functions/vshnpostgres/alerts.go

```diff
@@ -0,0 +1,25 @@
+package vshnpostgres
+
+import (
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/functions/common/nonsla"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+var maxConnectionsAlert = promv1.Rule{
+	Alert: "PostgreSQLConnectionsCritical",
```

Comment:
Yeah, as you can see I remove it for several reasons:
* It's redundant information, the description of the alert contains more in-depth information
* It's the container name which can be very confusing for the customer, they have a PostgreSQL instance and now get errors about patroni something
* The generic alerts now also omit the container name, so that are actually generic